examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -327,7 +338,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.6.8"
```

Comment:
What's the reason roll back to 3.6.8?

Reply:
this is just jupyter keeping track of the python version sadly...